### DISCUSSION TITLE
퍼블리시 업로드 오류

### DISCUSSION THREAD

**Discussion by uniconnect147** [naverz/zepeto-studio-kor#1094] [5. Q&A]
맵을 다만들고 퍼블리시하고 업로드했더니 아래와 같은 오류가 뜨는데요
![image](https://user-images.githubusercontent.com/109579358/181398303-8bcdadca-fd02-4cad-8696-7d9c75055cb3.png)

어떻게 해결해야할까요??

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 패키지 생성 이후에 뜬 오류라서 스튜디오에는 정상적으로 업로드가 되어 있는지 확인이 필요해 보입니다.
만약 업로드가 안되어 있다면 Publish 후 패키지 파일을 다운로드 받는 방법으로 저장해 보세요.
그 후 직접 스튜디오에 업로드 해보시고 문제 발생시 알려주세요~
